content/testsuite/testsuite.go

```diff
@@ -329,7 +330,10 @@ func checkInfo(ctx context.Context, cs content.Store, d digest.Digest, expected
 	if info.CreatedAt.After(c2) || info.CreatedAt.Before(c1) {
 		return errors.Errorf("unexpected created at time %s, expected between %s and %s", info.CreatedAt, c1, c2)
 	}
-	if info.UpdatedAt.After(u2) || info.UpdatedAt.Before(u1) {
+	// FIXME: broken on windows: unexpected updated at time 2017-11-14 13:43:22.178013 -0800 PST,
```

Comment:
There is something really odd going on here: was having the same problem in #1708.